### ISSUE TITLE
Spin Boxes do not announce their current value

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32175]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When you enter a spin button by either pressing enter on a button or arrowing on to it (note there seem to be two different types and ways of handling these being used in MuseScore currently) they do not announce their current value.  The user has to arrow down and up to hear the current setting.  In some cases as navigation seems to be possible in all 4 directions and confusing it is hard to even know what the current setting is.  See 32171 

Note also that the below are just two examples, but this effects most if not all spin boxes throughout the software that do not report their current setting when you enter them.

STEPS:
1.  Shift Tab several times to get to the Playback area then left arrow on to the speed change button.
2. Press Enter to Open it.  Note it does not announce anything.
3. press down then up arrow to hear the current setting.  Related 31480 
4. Press Esc several times till you get back to the score.
5. Alt arrow to the tempo mark and press F2.
6. Press Tab to move on to the Text Style Setting area.
7. Press right arrow and you hear you are in the font size spin box.  However, it does not announce the current value and this one seems to navigate unexpectedly so down then up is unclear what the current value was when you entered this area, and left/right arrows seem to alter it also.

There are many of these in other areas including Properties 16739 Ideally all spin boxes should announce their value on entry.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
